packages/beyondinsight_password_safe/data_stream/session/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,212 @@
+---
+description: Pipeline for processing BeyondInsight and Password Safe Sessions logs
+processors:
+  - rename:
+      field: message
+      target_field: beyondinsight_password_safe.session
+  - drop:
+      if: ctx.error?.statuscode != null && ctx.error.statuscode != 200
+  - fingerprint:
+      fields:
+        - beyondinsight_password_safe.session.SessionID
```

```suggestion
        - beyondinsight_password_safe.session
```

To index changes when they happen.